app/javascript/components/pages/Home.js

```diff
@@ -1,14 +1,64 @@
 import React, { Component } from "react";
-import { Button } from 'reactstrap'
+import { Button, Jumbotron, Card, CardTitle, Col, CardText, Media } from 'reactstrap'
```

Comment:
Is Media being used?

Reply:
It will be. Tried to add an image but wasn't working and not necessary to MVP. So that's an icebox feature. I'll remove it from the import